### PR TITLE
LPS-156523 Thank you page options

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Review by sandrodw3 — COMMENTED** [review]
Super cool! 🥰 Only left a chorricomment

**Comment by victorg1991** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FormGeneralPanel.js [hunk 1]
```diff
@@ -144,7 +153,211 @@ function FormOptions({item, onValueSelect}) {
...
+			Liferay.Language.get(
+				'thank-you.-your-information-was-successfully-received'
+			)
+		)
+	);
+	const [url, setUrl] = useControlledState(
+		getEditableLocalizedValue(successMessageConfig.message, languageId)
+	);
+	const [showMessagePreview, setShowMessagePreview] = useControlledState(
+		item.config.showMessagePreview
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FormGeneralPanel.js]: Fixed!!

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**